app/views/admin/reports/_date_range_form.html.haml

```diff
@@ -6,8 +6,7 @@
 
 .row.date-range-filter
   .alpha.two.columns= label_tag nil, t(:date_range)
-  .omega.fourteen.columns
-    = f.text_field "#{field}_gt", :class => 'datetimepicker datepicker-from', :placeholder => t(:start), data: { controller: "flatpickr", "flatpickr-enable-time-value": true, "flatpickr-default-date-value": "startOfDay" }, value: start_date 
-    %span.range-divider
-      %i.icon-arrow-right
-    = f.text_field "#{field}_lt", :class => 'datetimepicker datepicker-to', :placeholder => t(:stop), data: { controller: "flatpickr", "flatpickr-enable-time-value": true, "flatpickr-default-date-value": "endOfDay" }, value: end_date 
+  .omega.fourteen.columns{ data: { controller: "flatpickr", "flatpickr-mode-value": "range" } }
```

Comment:
Could you use the "date_range" partial instead ? 
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/views/admin/reports/_date_range_form.html.haml

Reply:
Hello @rioug, 

I don't understand, the link displayed goes to the very file I have modified, which is a partial.

Reply:
:facepalm: sorry, for some reason I though you copied the flat picker code from the partial when in fact you updated the partial. please ignore.